syntax/crystal.vim

```diff
@@ -272,7 +272,8 @@ syn match crystalLinkAttr "]" contained containedin=crystalLinkAttrRegion displa
 if !exists('g:crystal_no_special_methods')
   syn keyword crystalAccess    protected private
   " attr is a common variable name
-  syn keyword crystalAttribute getter setter property class_getter class_setter class_property abstract
+  syn keyword crystalAttribute abstract
+  syn match   crystalAttribute "\<\%(class_\)\=\%(getter\|setter\|property\)\ze[!?]\= " display
```

Comment:
I think this `\ze` is not necessary because we want to contain `?` and `!` in highlights. And `\s` would be better for spaces (though 99% people would use space rather than tab for writing Crystal):

```vim
syn match   crystalAttribute "\<\%(class_\)\=\%(getter\|setter\|property\)[!?]\=\s" display
```

Reply:
Hmm, agree.  :thinking: